typescript/test/unit/baselines.ts

```diff
@@ -45,7 +45,7 @@ describe('Baseline tests', () => {
     outputDir: '.test-out-kms',
     protoPath: 'google/cloud/kms/v1/*.proto',
     useCommonProto: false,
-    iamService: true,
+    serviceYaml: 'google/cloud/kms/v1/cloudkms_v1.yaml',
```

Comment:
The fact that the KMS baseline tests did not change basically means that your fixes are correct. There is no need to add IAM to Showcase.